Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   gem "pry"
```

Comment:
pry-byebug was to make it play nicely with byebug (a lot of the community seems big on byebug. Tricia uses it pretty heavily).

pry-remote is/was for connecting a pry-repl to ruby processes running under passenger. Useful if you were using the Vagrant dev environment, probably less useful now.